torchbenchmark/models/dlrm/__init__.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, device=None, jit=False):
 
         # Preparing data
         X, lS_o, lS_i, self.targets = next(iter(self.train_ld))
-        if self.device == "cuda":
+        if self.device != "cpu":
```

Comment:
One question, can we just delete this condition? I guess if `self.device == "cpu"`, `X = X.to(self.device)` is just no-op?

Reply:
Yea, seems even better.